src/gobexport/test.py

```diff
@@ -114,34 +114,37 @@ def test(catalogue):
 
             for filename in filenames:
                 # Check the previously exported file at its temporary location
-                obj_info, obj = _get_file(conn_info, f"{EXPORT_DIR}/{catalogue}/{filename}")
-
-                # Clone check so that changes to the check file don't affect other runs
-                check = copy.deepcopy(_get_check(checks, filename))
-
-                # Report results with the name of the matched file
-                matched_filename = obj_info['name'] if obj_info else filename
+                obj_info, obj_contents = _get_file(conn_info, f"{EXPORT_DIR}/{catalogue}/{filename}")
 
                 if obj_info is None:
                     logger.error(f"File {filename} MISSING")
-                elif check:
-                    stats = _get_analysis(obj_info, obj, check)
-                    if _check_file(check, matched_filename, stats):
+                    continue
+
+                # Clone check so that changes to the check file don't affect other runs
+                if file_checks := copy.deepcopy(_get_check(checks, filename)):
```

Comment:
Make this "if not condition: continue" as well then.

<3 guards/early returns over nested if/else conditions.

Reply:
But we don't need to continue.. We execute the "else" block and still need to write the proposal